handbook/product/roadmap.md

```diff
@@ -55,11 +55,14 @@ section Web
 
 1. **Customer outreach to improve adoption of campaigns**
 1. User credentials ([RFC 242](https://docs.google.com/document/d/1SqoWWm1xs82QibrWwYsXmpmgweN6EpcKt1qXrRBjjlU/edit)), which will allow non-site-admins to create campaigns
+1. Better burndown charts
 1. Versioning/releasing of src-cli with respect to sg/sg
-1. Respect GitHub GraphQL rate limits
 1. Improved documentation of src-cli login process
+1. Add filtering/searching to campaign and changeset lists
+1. Ability to auto-merge changesets, depending on various conditions
+1. Ability to add default reviewers to changesets
```

Comment:
Maybe these two shouldn't be on the same level as the others as I think it's not yet clear what these two involve and they might be pushed back if too much effort(?)

Reply:
While that is true, these are currently our best guess as to what we want to tackle next, and I wanted this to be as accurate as we currently know. As we learn more (and possibly push one or both of these back), I will keep this list updated.